src/HPWH.in.hh

```diff
@@ -475,6 +486,12 @@ class HPWH {
   double member_inletT_C;
   double member_minutesPerStep;
 
+  bool doInversionMixing;
+  /**<  whether or not to model will mix temperature inversions in the tank  */
+
```

Comment:
Comments of this sort are ambiguous.  Something like the following is better.
// iff true, model inversion mixing <whatever>

where iff = if and only if